src/corefx/System.Globalization.Native/calendarData.cpp

```diff
@@ -122,24 +123,27 @@ const char* GetCalendarName(CalendarId calendarId)
 		case JAPANESELUNISOLAR:
 		case TAIWANLUNISOLAR:
 			return "chinese";
```

Comment:
ICU/CLDR [supports](http://userguide.icu-project.org/datetime/calendar) the following calendars:
Gregorian
Japanese
Buddhist
Chinese
Persian
Indian
Islamic
Hebrew
Indian
Coptic
Ethiopic

The "Chinese" calendar in CLDR seems to represent all 4 of these calendars.